dask/array/stats.py

```diff
@@ -228,12 +228,14 @@ def skewtest(a, axis=0, nan_policy="propagate"):
         )
 
     b2 = skew(a, axis)
-    n = float(a.shape[axis])
-    if n < 8:
+    n = _lazy_shape(a, axis)
+    if a.shape[axis] < 8:
```

Comment:
In the case where we need `_lazy_shape`, `a.shape[axis] < 8` will always evaluate to False. 

This PR doesn't make that condition any _worse_, but thought I'd point it out in case someone has good idea how to fix that.